pyo3cls/src/lib.rs

```diff
@@ -83,8 +83,9 @@ pub fn pyfunction(attr: TokenStream, input: TokenStream) -> TokenStream {
     let mut ast = parse_macro_input!(input as syn::ItemFn);
     let args = parse_macro_input!(attr as PyFunctionAttr);
 
+    // TODO: Support #[name = "..."] attribute?
```

Comment:
This: should `#[pyfunction]` also support this attribute? I think it can be done with a little more refactoring.

Reply:
I think supporting `#[pyfunction]` would be a good idea!